packages/components/src/chart/d3chart/utils/line-chart.js

```diff
@@ -98,7 +101,8 @@ export const drawLines = ( node, data, params, scales, formats, tooltip ) => {
 		.attr( 'class', 'line-g' )
 		.attr( 'role', 'region' )
 		.attr( 'aria-label', d => d.label || d.key );
-	const dateSpaces = getDateSpaces( data, params.uniqueDates, width, scales.xScale );
+	const reversedKeys = params.visibleKeys.slice().reverse();
```

Comment:
Will keys always need to be reversed when using this function?  Should we move this inside `getDateSpaces()` or is there a case where keys would already be in the order we need?

Reply:
Absolutely! It makes more sense to move it inside `getDateSpaces()`.

Done in 95f8b2e38d6efffcbc91984ef753b31f23d57927.